examples/mcp/shared/apps/items.py

```
@@ -0,0 +1,169 @@
+
+@app.get(
+    "/items/search/",
+    response_model=List[Item],
+    tags=["search"],
+    operation_id="search_items",
+)
+async def search_items(
+    q: Optional[str] = Query(None, description="Search query string"),
+    min_price: Optional[float] = Query(None, description="Minimum price"),
+    max_price: Optional[float] = Query(None, description="Maximum price"),
+    tags: List[str] = Query([], description="Filter by tags"),
```

Ruff (0.12.2)

Do not perform function call `Query` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)